sdk/resource/iterator_test.go

```diff
@@ -12,21 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package resource
```

Comment:
Could probably just remove this file now that there is a unified place (not here) that implements (and tests) the iterator. These are just testing that implementation anyways.

Reply:
I moved this test into the api/label package.